src/entities/course/ui/course-card/course-card.module.scss

```diff
@@ -0,0 +1,97 @@
+.course-card {
+  $accent-color: var(--accent-bg-color);
+
+  @extend %transition-all;
+
+  display: flex;
+  flex-direction: column;
+
+  width: 380px;
+  min-width: 100px;
+  height: 100%;
+
+  border-radius: 12px;
+
+  .card-header {
+    position: relative;
+
+    overflow: hidden;
+    display: flex;
+    gap: 24px;
+    align-items: center;
+
+    height: 100%;
+    min-height: 112px;
+    padding: 0 24px;
+
+    background-color: $color-purple-50;
+    border: solid $color-gray-200;
+    border-width: 1px 1px 0 1px;
+    border-radius: 12px 12px 0 0;
+
+    &::after {
+      @extend %transition-all;
+
+      @include card-corner-accent($accent-color);
+    }
+
+    h3 {
+      margin: 0;
+    }
+
+    img {
+      width: 60px;
+      height: 54px;
+    }
+  }
+
+  .card-title {
+    width: 100%;
+
+    font-size: 24px;
+    font-weight: $font-weight-medium;
+    line-height: 32px;
+    color: $color-gray-600;
+
+    @include media-laptop {
+      font-size: 20px;
+    }
+
+    @include media-tablet {
+      font-size: 24px;
+    }
+  }
+
+  .course-info {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    padding: 24px;
+
+    background-color: hsla(from $color-gray-100 h s l/ $opacity-20);
+    border: solid $color-gray-200;
+    border-width: 0 1px 1px 1px;
+    border-radius: 0 0 12px 12px;
+  }
+
+  &:hover .card-header::after {
+    @include card-corner-accent($accent-color, $isHovered: true);
+  }
```

Comment:
```suggestion
@include media-hover{
    &:hover .card-header::after {
      @include card-corner-accent($accent-color, $isHovered: true);
    }
  }
```

Reply:
Actually, I don't know why we need it. I tried to add my mixin to `@include media-hover` but it didn't work. But now it's ok and I fixed it